CRM/Contribute/Form/Search.php

```diff
@@ -152,6 +152,16 @@ public function preProcess() {
    * @throws \Exception
    */
   public function setDefaultValues() {
+    $lowReceiveDate = CRM_Utils_Request::retrieve('start', 'Timestamp');
+    if (!empty($lowReceiveDate)) {
+      $this->_formValues['receive_date_low'] = date('Y-m-d H:i:s', strtotime($lowReceiveDate));
+      CRM_Core_Error::deprecatedFunctionWarning('pass receive_date_low not start');
+    }
+    $highReceiveDate = CRM_Utils_Request::retrieve('end', 'Timestamp');
+    if (!empty($highReceiveDate)) {
+      $this->_formValues['receive_date_high'] = date('Y-m-d H:i:s', strtotime($highReceiveDate));
+      CRM_Core_Error::deprecatedFunctionWarning('pass receive_date_high not end');
+    }
```

Comment:
@eileenmcnaughton do we need to port the relative_handling as well?

Reply:
@seamuslee001 so I think the relative date handling is not handling the relative date in the url but ensuring it is zero'd if the high or low are set - which I think IS handled in the generic handling